src/translation/translations/en.json

```diff
@@ -529,5 +529,9 @@
     "dailyDistribution": "Daily Distribution:",
     "progressBar": "Xvs Distribution progress",
     "remaining": "Remaining:"
+  },
+  "pagination": {
+    "itemOf": "item {{currentPageLastIndex}} out of {{itemsCount}}",
+    "itemsOf": "items {{firstItemNumber}} - {{currentPageLastIndex}} out of {{itemsCount}}"
```

Comment:
@coreyar I'd like your opinion on this one. I asked Dmitriy to write the content this way instead of like in the designs.
The designs write "25 - 36 items of 150" but this seems incorrect to me in English. I suggested Dmitriy to write "items 25 - 36 out of 150" instead, but I'm also not 100% sure on that.
What do you think?

Reply:
They both seem fine to me. I think the way you have it works.  `out of ` sounds better